Source/Blazorise/Licensing/BlazoriseLicenseProvider.cs

```diff
@@ -43,88 +43,84 @@ public sealed class BlazoriseLicenseProvider
     private int? limitsListViewMaxRows;
 
     private int? limitsTreeViewMaxRows;
-
     #endregion
 
     #region Constructors
-
     /// <summary>
     /// A default <see cref="BlazoriseLicenseProvider"/> constructor.
     /// </summary>
     /// <param name="options"></param>
     /// <param name="jsRuntime"></param>
     /// <param name="versionProvider"></param>
-    public BlazoriseLicenseProvider( BlazoriseOptions options, IJSRuntime jsRuntime, IVersionProvider versionProvider )
+    public BlazoriseLicenseProvider(BlazoriseOptions options, IJSRuntime jsRuntime, IVersionProvider versionProvider)
     {
         this.options = options;
         this.jsRuntime = jsRuntime;
         this.versionProvider = versionProvider;
 
-        if ( !initialized )
+        if (!initialized)
         {
             backgroundWorker = new BackgroundWorker();
             backgroundWorker.DoWork += BackgroundWorker_DoWork;
             backgroundWorker.RunWorkerAsync();
         }
     }
-
     #endregion
 
     #region Methods
-
-    private async void BackgroundWorker_DoWork( object sender, DoWorkEventArgs e )
+    private async void BackgroundWorker_DoWork(object sender, DoWorkEventArgs e)
```

Comment:
Please fix the space formatting. The format settings should be picked by the `.editorconfig` automatically.

Reply:
fixed, wasn't turned on..

Reply:
it figured out "new" stuff, which I am not sure if goes from editor config or my ide setup, please re-check that.